lib/reverse_sentence.rb

```diff
@@ -0,0 +1,4 @@
+# A method to reverse the words in a sentence, in place.
+def reverse_sentence(my_sentence)
+  my_sentence << "NOT IMPLEMENTED"
```

Comment:
See [my comment](https://github.com/AdaGold/reverse_words/pull/1/files#r160554004) on the `reverse_words` repo, which applies equally here.